### ISSUE TITLE
fail to recognize utf-8-bom

### ISSUE THREAD

**Issue by hax** [jedmao/eclint#40]
``` sh
echo test > test
eclint fix -c utf-8-bom test
eclint check -c utf-8-bom test
```

output:

```
test: expected charset: utf-8-bom
```

It seems `eclint fix` add bom successfully, but `eclint check` fail to recognize it... :fearful:

**Comment by hax** [comment]
eclint version: 1.1.5
node version: 5.8.0

**Comment by jedmao** [comment]
Can you provide a failing test? Because there is already a test for this here: https://github.com/jedmao/eclint/blob/master/lib/rules/charset.spec.ts#L72-L77

**Comment by hax** [comment]
I already give you the test (3 lines shell script) in my first comment. You can just run it.

I know you have the testcase, but unfortunately the feature is still broken, and I don't know why.

**Comment by gucong3000** [comment]
@hax fixed, try `1.1.5`
